src/backend/web/handlers/webcasts.py

```diff
@@ -0,0 +1,24 @@
+from datetime import datetime, timedelta
+
+from werkzeug.wrappers import Response
+
+from backend.common.decorators import cached_public
+from backend.common.flask_cache import make_cached_response
+from backend.common.models.event import Event
+from backend.web.profiled_render import render_template
+
+
+@cached_public
+def webcast_list() -> Response:
+    year = datetime.now().year
+    events = Event.query(Event.year == year).order(Event.start_date).fetch(500)
```

Comment:
This should use our cached queries if possible https://github.com/the-blue-alliance/the-blue-alliance/blob/0a59f9b3a619ee0ebcb0097d9bf78109a61efd2a/src/backend/common/queries/event_query.py#L34

Reply:
thanks for the tip, updated!